### PR TITLE
Optimize hack/local-up-kubeedge.sh

### PR DESCRIPTION
**Pull request by llhuii**
**What type of PR is this?**

/kind bug
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by daixiang0** [comment]
Please retitle with a more meaningful title, like `hack/local-up-kubeedge.sh: fix xxx`

**Comment by llhuii** [comment]
> Please retitle with a more meaningful title, like `hack/local-up-kubeedge.sh: fix xxx`

There are 3 commits in this pr, don't know how to say `xxx`. 

May I submit them in three separated pull requests?

**Comment by daixiang0** [comment]
"optimize hack/local-up-kubeedge.sh"


lgtm

File: hack/local-up-kubeedge.sh [hunk 1]
```diff
@@ -121,6 +123,7 @@ function start_edgecore {
   sed -i -e "s|token: .*|token: ${token}|g" \
       -e "s|hostnameOverride: .*|hostnameOverride: edge-node|g" \
       -e "s|/etc/|/tmp/etc/|g" \
+      -e "s|/var/lib/kubeedge/|/tmp&|g" \
```
Comment (fisherxu) [inline hack/local-up-kubeedge.sh:126]: What's the meaning of `/tmp&`, for files under `/var/lib/kubeedge`, i'd prefer keep them here, not move to /tmp.

Comment (llhuii) [reply hack/local-up-kubeedge.sh:126]: > What's the meaning of `/tmp&`?

'&' means the whole matched string, in this context i.e. `/var/lib/kubeedge/`. 

>  For files under `/var/lib/kubeedge`, i'd prefer keep them here, not move to /tmp.
1. What's the difference between `/etc` and `/var/lib/kubeedge`? Why one is under `/tmp`, the other is not?
2. https://github.com/kubeedge/kubeedge/blob/29c26a61b3541331ae51917df83cbf6adcd7ea62/hack/local-up-kubeedge.sh#L48 needs to be changed if `/var/lib/kubeedge` keeps unchanged.

Comment (llhuii) [reply hack/local-up-kubeedge.sh:126]: Better to move all related files into a directory, also I can make it configurable if you wants.

Comment (fisherxu) [reply hack/local-up-kubeedge.sh:126]: > &' means the whole matched string, in this context i.e. /var/lib/kubeedge/.

Get it, also fine to put under tmp.

**Pull request merged**